webapp/src/components/react_select_setting.jsx

```diff
@@ -83,8 +91,9 @@ export default class ReactSelectSetting extends React.PureComponent {
             >
                 <ReactSelect
                     {...this.props}
-                    styles={this.props.theme && this.getStyle(this.props.theme)}
+                    menuPortalTarget={document.body}
                     onChange={this.handleChange}
+                    styles={this.getStyle(this.props.theme)}
```

Comment:
I think the null check belongs in the `getStyle` function above, as I noted earlier, so it's a good change to remove it from here. 👍